.github/workflows/tests.yml

```
@@ -23,4 +23,4 @@ jobs:
           github.event_name == 'pull_request'
           && !contains(github.event.pull_request.labels.*.name, 'ci:run-full-test-suite')
           || false
-         }}
+         }}
```

What's this change about? Can we revert so this file isn't updated as part of this PR?
```suggestion
         }}

```